backend/studyagent/taskmaster/donor/onboarding.py

```
@@ -0,0 +1,186 @@
+
+    return config
+
+
+def load_config() -> dict:
+    """Load saved config, or sensible defaults if onboarding hasn't run."""
+    try:
+        value = load_config_dict()
+        if value.get("priority_mode"):
+            return value
+    except Exception:
+        pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)